app/controllers/pages_controller.rb

```diff
@@ -1,4 +1,6 @@
 class PagesController < ApplicationController
+  rescue_from ActionView::MissingTemplate, with: :rescue_missing_template
```

Comment:
Not sure if this has a request spec around it already, but may be worth adding if you're refactoring it?

Reply:
Think its already covered but will check

Reply:
Wasn't covered so added a spec